src/RestaurantButton.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+function RestaurantButton({ eventHandler }) {
```

Comment:
Again same as before - think of a different name for this function you're destructuring